docs/modules/ROOT/pages/create-github-action.adoc

```diff
@@ -338,18 +338,20 @@ The token is required when consuming the GitHub REST or GraphQL APIs so, in most
 [[private_repo]]
 [WARNING]
 ====
-The workflow above will malfunction when used in a private repository. The automatically generated `GITHUB_TOKEN` only grants access to packages, that are part of the https://docs.github.com/en/packages/learn-github-packages/about-permissions-for-github-packages#access-tokens[same repository] the workflow is defined in.
+You may call a Quarkus GitHub Action defined in a public repository from a public or private GitHub repository.
 
-To use a _GitHub Action_ defined in a separate private repository, belonging to the same user or organization, please follow these steps:
+However, the workflow above will malfunction when the Quarkus GitHub Action is hosted in a private repository as the GitHub hosted Maven repository might not be accessible.
+The automatically generated `GITHUB_TOKEN` only grants access to private packages that are part of the https://docs.github.com/en/packages/learn-github-packages/about-permissions-for-github-packages#access-tokens[same repository] the workflow is defined in.
 
-- Create or use a https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens[Personal Access Token] with `read:packages` scope. Depending on the actions to be performed, additional permissions can be necessary.
+To use a Quarkus GitHub Action hosted in a separate private repository, please follow these steps:
 
+- Create or use a https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens[Personal Access Token] with the `read:packages` scope. Depending on the GitHub API calls to be performed, additional permissions can be necessary.
 - In the repository, that is using the action, https://docs.github.com/en/actions/security-for-github-actions/security-guides/using-secrets-in-github-actions[create a secret] using the token.
+- The name you give to the secret (e.g. `MY_ACTION_TOKEN`) must in turn be referenced in the workflow configuration above: Replace &#x2780; with `github-token: ${{ secrets.MY_ACTION_TOKEN }}`.
 
-- The name you give the secret (e.g. `MY_ACTION_TOKEN`) must in turn be referenced in the workflow configuration above: Replace  &#x2780; with `github&#8209;token:&nbsp;${{&nbsp;secrets.MY_ACTION_TOKEN&nbsp;}}`.
```

Comment:
Here I want to avoid the special characters in case people are copy/pasting the content to their workflow (even if I agree avoiding a new line would have been nice).

Reply:
For me copying and pasting the text containing the special characters presents no problem.
To guarantee compatibility with all OSs and browsers, I will remove them.